src/Backend/Modules/MediaLibrary/Resources/views/BackendMediaLibraryUpload.html.twig

```diff
@@ -2,9 +2,6 @@
     ====================================================================== -->
 <script type="text/template" id="qq-template-gallery">
   <div class="qq-uploader-selector qq-uploader qq-gallery" qq-drop-area-text="{{ 'lbl.DropFilesHere'|trans|ucfirst }}">
-    <div class="qq-total-progress-bar-container-selector qq-total-progress-bar-container">
```

Comment:
Why did you remove the progress bar?

Reply:
It didn't show half of the time and was flashing, that made it look like something was wrong as stated in the commit message

Reply:
Hmmm it probably goes to fast if you upload smaller image.
But when uploading larger MB files that could come in handy?!

My suggestion is too not include this in your PR.
We can always remove it later.

Reply:
It needs to go or needs styling, but the current state is not acceptable, it was also not clear that it was a progressbar. I thought it was some sort of panel glitch

Reply:
@forkcms/frontenders @forkcms/backenders @forkcms/core-contributors  what do you think?

Reply:
I think we should remove the progress bar: If I upload a large image (7MB), I first see the placeholder image for some time en than the progress bar flashes really quick and disappears. So the progress bar seems kind of useless to me.

Also: selecting multiple images at once flashes the cropper popup for some reason, maybe that should be fixed too.

Reply:
This pr is fixing the flashing :)